web/src/components/spinner-button.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+import '../../css/spinner-button.css';
+
+
+const SpinnerButton = () => {
+  return (
+    <button className="spinnerButton spinning" disabled="disabled">Submit</button>
```

Comment:
https://github.com/Common-Voice/sentence-collector/pull/179#discussion_r259615176

I think this wasn't clear enough. What I meant was to rename the text content of the button, not the class in the CSS file, sorry about that.

```
<button className="spinnerButton spinning" disabled="disabled">Submitting...</button>
```

Reply:
Oh, I see know what you meant, you actually said label not class. Sorry, my fault!